lib/Image.php

```diff
@@ -40,18 +37,8 @@ public static function getInstance(): Adapter\GD|Adapter\Imagick|null
      *
      * @internal
      */
-    public static function create(): Adapter\GD|Adapter\Imagick|null
+    public static function create(): AdapterInterface
```

Comment:
@brusch The method is internal. So we can remove it? It was only used in the getInstance() method.

Reply:
Yeah, I thought about this as well, but was not sure if it's used somewhere in a bundle, but you're right, probably better to remove it and see 😉🤞